.github/workflows/RunTests.yaml

```diff
@@ -27,6 +27,36 @@ jobs:
     - name: Run tests
       run: .\Tests\TestRunner.ps1 -TestResults -Verbosity Normal
       shell: pwsh
+    - name: Publish test results
+      uses: EnricoMi/publish-unit-test-result-action/composite@v1
+      continue-on-error: true
+      with:
+        files: testResults.xml
+
+  build-linux:
```

Comment:
(Couldn't leave a comment on a row you didn't change so I'm adding it here) - Can you update the name for the tests on windows from `build` to `build-windows`?